src/clients/victoriaMetrics/metricFunctions.ts

```diff
@@ -0,0 +1,45 @@
+export function validateLabelNamesAndOptions(
```

Comment:
What you're looking for here is set difference. 

Convert both lists to sets, make sure their lengths before conversion is same, then do an equals operation on the sets.

Reply:
Can you also add a comment above this function to specify what is a "valid" label name / options combo.